pkg/pipeline/builder/websocket.go

```diff
@@ -33,5 +34,9 @@ func BuildWebsocketBin(pipeline *gstreamer.Pipeline, appSinkCallbacks *app.SinkC
 		return nil, err
 	}
 
+	b.SetGetSrcPad(func(name string) *gst.Pad {
```

Comment:
this probably isn't needed since the sink will only have one template

Reply:
Removed